python/ql/src/Security/CWE-327/examples/broken_crypto.py

```diff
@@ -6,7 +6,7 @@ def send_encrypted(channel, message):
     channel.send(cipher.encrypt(message)) # BAD: weak encryption
 
 
-cipher = Blowfish.new(SECRET_KEY)
+cipher = AES.new(SECRET_KEY)
```

Comment:
For consistency, you should change `Blowfish` to `AES` in the `import` statement in the first line of this file.

Reply:
Thanks @tausbn 👍🏼 😃

Reply:
I'm slightly confused. Line one appears unchanged to me? :confused:

Reply:
oops, sorry